submissions/zhenyakornilov/js-dom/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" id="root">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/269a1f5e0e.js" crossorigin="anonymous"></script>
+  <link rel="icon" type="image/x-icon" href="images/museum-icon.svg">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Top museums of Kyiv</title>
+</head>
+<body class="body">
+  <div class="wrapper">
+    <header class="header">
+      <span class="dropdown-wrap">
+        <button class="dropdown">
+          <i class="fa-solid fa-bars fa-lg"></i>
```

Comment:
fa-solid fa-bars fa-lg. What are these classes for?

Reply:
It's Font Awesome prebuilt classes, I thought it is not needed to change them.

Reply:
you are right. my bad